packages/module/patternfly-docs/content/extensions/chatbot/examples/Messages/Messages.md

```diff
@@ -79,7 +79,7 @@ You can add actions to a message, to allow users to interact with the message co
 
 ### Custom message actions
 
-Beyond the standard message actions (positive, negative, copy, share, or listen), you can add custom actions to a bot message by passing an `actions` object to the `<Message>` component. This object can contain the following customizations: `ariaLabel`, `onClick`, `className`, `isDisabled`, `tooltipContent`, `tooltipProps`, and `icon`.
+Beyond the standard message actions (positive, negative, copy, share, or listen), you can add custom actions to a bot message by passing an `actions` object to the `<Message>` component. This object can contain the following customizations: `ariaLabel`, `clickedAriaLabel`, `onClick`, `className`, `isDisabled`, `tooltipContent`, `tooltipContent`, `tooltipProps`, and `icon`. `clickedAriaLabel` and `clickedTooltipContent` are applied only when a button is clicked. If `clickedAriaLabel` or `clickedTooltipContent` are omitted, they will default to the `ariaLabel` or `tooltipContent` supplied.
```

Comment:
```suggestion
Beyond the standard message actions (good response, bad response, copy, share, or listen), you can add custom actions to a bot message by passing an `actions` object to the `<Message>` component. This object can contain the following customizations: 
- `ariaLabel`
- `onClick`
- `className`
- `isDisabled`
- `tooltipContent`
- `tooltipContent`
- `tooltipProps`
- `icon` 

You can apply a `clickedAriaLabel` and `clickedTooltipContent` once a button is clicked. If either of these props are omitted, their values will default to the `ariaLabel` or `tooltipContent` supplied.
```
do you think changing this to a list is more digestible? 

Another idea is to remove the list of props and just link to the props table, since there's more context for each of these props down there? I think I lean to this solution if you think it's okay, but I'll let you choose between the 2!

> Beyond the standard message actions (good response, bad response, copy, share, or listen), you can add custom actions to a bot message via the [`actions` object props](#actionprops). 
> 
> You can apply a `clickedAriaLabel` and `clickedTooltipContent` once a button is clicked. If either of these props are omitted, their values will default to the `ariaLabel` or `tooltipContent` supplied.

Reply:
I'd love to do option 2, but we have a problem with the docs framework right now where if you click an anchor link, it doesn't take you there, and I hate it. If it's ok with you, I'll go with the first one for now and we can totally change it later.

Reply:
Ahh do we? That is a bother 😭 I think the 1st approach is still fine!